Resources/Prototypes/_Moffstation/AppearanceCustomization/station_ai.yml

```
@@ -20,7 +19,7 @@
     Occupied:
       sprite: _Moffstation/Mobs/Silicon/station_ai.rsi
       state: ai_angryface
-    Dead: *defaultdead
+    Dead: &defaultdead
```

```suggestion
    Dead: &defaultdead
      sprite: Mobs/Silicon/station_ai.rsi
      state: ai_dead
```
Need to move the content of `defaultdead` to this one if this is the new definition of the anchor.